index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Delis videography</title>
+
+  <link rel="stylesheet" href="./stylesheets/index.css">
+  <script src="https://kit.fontawesome.com/8d20eebc7a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <section class="topHeader">
+      <ul>
+        <li><i class="fab fa-facebook-square fa-lg"></i></li>
+        <li><i class="fab fa-twitter-square fa-lg"></i></li>
+        <li><p class="headText">English</p></li>
+        <li><p class="headText">My Page</p></li>
+        <li><p class="headText">Logout</p></li>
+      </ul>
+    </section>
+
+    <section class="header">
+      <a href="./index.html" class="logo">
+        <img src="./assets/logo1.png" alt="logo">
+      </a>
+  
+      <!-- ============ Navigation ===================== -->
+      <nav>
+        <div class="nav-icon" onclick="toggleMenu()">
+          <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
+        </div>
+  
+        <div class="nav-links-container">
```

Comment:
- Kindly as was requested by the previous reviewer, move the hamburger button on your mobile version to the left. And plus, remove the logo icon from your mobile version to match with Behance design

![image](https://github.com/Izebiafe/videography-capstone/assets/93630700/6806d80e-3751-49a3-8d59-c725f8ac8bb0)